deploy/001-risk-fund-converter.ts

```diff
@@ -92,15 +126,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     poolGameFi,
     poolLiquidStakedBNB,
     poolTron,
-    USDT,
+    usdt,
+    btcb,
+    eth,
     vBNB,
     wBNB,
     proxyOwnerAddress,
   } = addresses[network.name];
 
   comptrollers = [corePool, poolStableCoin, poolDeFi, poolGameFi, poolLiquidStakedBNB, poolTron];
-  assets = [[USDT], [USDT], [USDT], [USDT], [USDT], [USDT]];
-  values = [[true], [true], [true], [true], [true], [true]];
+  assets = [[usdt, btcb, eth], [usdt], [usdt], [usdt], [usdt], [usdt]];
+  values = [[true, true, true], [true], [true], [true], [true], [true]];
```

Comment:
We don't have USDT in the `poolLiquidStakedBNB`

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/48/commits/8ef869acea87739304b75a35b79cacc152133db7